bottlecap/src/lifecycle/invocation/triggers/body.rs

```diff
@@ -0,0 +1,32 @@
+use std::io::{Cursor, Read};
+
+use base64::Engine;
+use bytes::{Buf, Bytes};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Default, Serialize, Deserialize, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Body {
```

Comment:
Body can be up to 6MB in AWS Lambda, seems like span inferring doesn't actually require this body in order to provide its work, is there a way this could be optional?

Reply:
I wanna say I can look into making the `body` field be a `Cow` on the input data; but the reality is this is likely containing JSON which means it'll need `\`-escapes decoded and will be a copy in any case...

It seems to me the only way to make the body optional (and decide on this at run time) would be to effectively duplicate all those structures... :/

Reply:
So what if we skip deserialization on `Body`, and then, depending on the method we use, let's say (from_slice), the body does get set? Is that even possible? Just throwing an idea

Reply:
The body will be processed by `serde_json` no matter what I believe... Either we get it out and "store" it, or we discard it right away.